weekend-bowlling/src/game.js

```diff
@@ -0,0 +1,38 @@
+'use strict'
+function Game() {
+  this.totalScore = 0;
+  this.allFrames = []
+  this.frame = []
+  this.bonus = {}
```

Comment:
What was your plan for the key value pair in this hash? Would the key be the frame and the value the score?